pkg/download/download_writer.go

```diff
@@ -53,21 +53,42 @@ func WriteToDisk(fs afero.Fs, writerContext WriterContext) error {
 }
 
 func writeToDisk(fs afero.Fs, writerContext WriterContext) error {
-
 	log.Debug("Preparing downloaded data for persisting")
 
-	manifestName := getManifestFilePath(fs, writerContext)
-	m := createManifest(writerContext)
+	manifestFilePath := getManifestFileName(fs, writerContext)
+	projectFolderPath := getProjectFolderName(fs, writerContext)
+
+	projectDefinition := manifest.ProjectDefinitionByProjectID{
+		writerContext.ProjectToWrite.Id: {
+			Name: writerContext.ProjectToWrite.Id,
+			Path: projectFolderPath,
+		},
+	}
+
+	manifest := manifest.Manifest{
+		Projects: projectDefinition,
+		Environments: map[string]manifest.EnvironmentDefinition{
+			writerContext.ProjectToWrite.Id: {
+				Name: writerContext.ProjectToWrite.Id,
+				URL: manifest.URLDefinition{
+					Type:  manifest.ValueURLType,
+					Value: writerContext.EnvironmentUrl,
+				},
+				Group: "default",
+				Auth:  writerContext.Auth,
+			},
+		},
+	}
 
 	outputFolder := writerContext.GetOutputFolderFilePath()
 
 	log.Debug("Persisting downloaded configurations")
 	errs := writer.WriteToDisk(&writer.WriterContext{
 		Fs:              fs,
 		OutputDir:       outputFolder,
-		ManifestName:    manifestName,
+		ManifestName:    manifestFilePath,
```

Comment:
nit: should the `WriterContext.ManifestName` also be renamed? If it's meant to be filepath, `ManifestName` is the wrong name for it

Reply:
actually..... it's just the name of a file I guess :D so `manifestFilePath` is wrong and should be called `manifestFileName` same for `projectFolderPath` which is also just the name of the folder.

Reply:
that brings me to the quick conclusion that "ProjectDefinition.Path" should be renamed to "ProjectDefinition.FolderName" :D it's confusing :D

Reply:
updated the PR pls take a look.
e2e tests are also green now :)